tests/test___init__.py

```diff
@@ -12,6 +12,7 @@
 import functools
 import itertools
 
+import mock
```

Comment:
@tseaver You should use `unittest.mock` (which means this would move to the stdlib imports section too). I usually use a [hack][1] for Python 2.7, so we can forget about it when 2020 comes:

```python
# conftest.py
if six.PY2:
    import mock

    unittest.mock = mock
    sys.modules["unittest.mock"] = unittest.mock
```

[1]: https://github.com/dhermes/bezier/blob/7c5b72e823cb8a88dbc81ea2581ba9e5fa204eba/tests/conftest.py#L27-L31